exrex.py

```diff
@@ -18,18 +18,24 @@
 #
 # (C) 2012- by Adam Tauber, <asciimoo@gmail.com>
 
-try:
-    from future_builtins import map, range
-except:
-    pass
-from re import match, sre_parse, U
+from sys import version_info
 from itertools import tee
 from random import choice, randint
 from types import GeneratorType
+from re import match, U
+
+IS_PY3 = version_info.major == 3
+IS_PY36_OR_GREATER = version_info[:2] >= (3, 6)
+IS_PY311_OR_GREATER = version_info[:2] >= (3, 11)
```

Comment:
No need for `[:2]`, Py is smart enough, I just re-tested cuz I've used the syntax with my apps no problem.